portfolio/src/main/webapp/style.css

```diff
@@ -147,12 +150,29 @@ label {
 input[type=submit] {
   display: block;
   margin: auto;
-  padding: 5px;
+  margin-top: 10px;;
 }
 
 #contact {
   background-color: aliceblue;
   border-radius: 25px;
   margin: 0 15% 0 15%;
   padding: 1% 10% 5% 10%;
+}
+
+button, 
```

Comment:
This looks like it will make the below styles apply to all `<button>` elements and can probably be removed. The `submit` id selector should be fine on its own.

Reply:
leaving this since i intended to apply this style to all the buttons